nimbus.fml.yaml

```diff
@@ -0,0 +1,29 @@
+---
+about:
+  description: The root level Feature Manifest for Focus for iOS
+  ios:
+    class: AppNimbus
+    module: Blockzilla
+channels:
+  - developer
+  - beta
+  - release
+features:
+  nimbus-validation:
+    description: A tiny feature to validate that Nimbus is working
+    variables:
+      bold-tip-title:
+        description: Make the tips title label bold
+        type: Boolean
+        default: true
+  onboarding-variables:
```

Comment:
This was called `onboading`, but it generated a class called `Onboarding` which had a collision with an existing class.